tests/js/client/shell/api/cursor.js

```diff
@@ -1212,9 +1212,225 @@ function dealing_with_cursorsSuite_fetching_floating_point_valuesSuite () {
   };
 }
 
+////////////////////////////////////////////////////////////////////////////////;
+// cursor with retriable request for the last batch;
+////////////////////////////////////////////////////////////////////////////////;
+
+function dealing_with_cursorsSuite_retriable_request_last_batch() {
+  let cn = "users";
+  return {
+    setUpAll: function () {
+      db._drop(cn);
+      let cid = db._create(cn);
+
+      let docs = [];
+      for (let i = 0; i < 2001; i++) {
+        docs.push({"_key": `test${i}`});
+      }
+      db[cn].save(docs);
+    },
+
+    tearDownAll: function () {
+      internal.debugClearFailAt();
+      db._drop(cn);
+    },
+
+    test_cursor_non_stream_request_retriable: function () {
+      let cmd = api;
+      let body = {"query": `FOR u IN ${cn} RETURN u`, "options": {"stream": true, "retriable": true}};
```

Comment:
The name of the test is `non_stream`, but the test performs a streaming query anyway. Looks wrong.